docs/paper/admin/reference/configuration/commands-configuration.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /reference/commands-configuration
```

Comment:
I’m not a fan of this slug / file name. This should indicate more that it is the outdated bukkit commands config rather than anything you need for plugins